setup

```diff
@@ -47,11 +47,37 @@ bash modules/openvpn/etc/scripts/platform/debian/setup.sh $ONBOARD_ROOT $ONBOARD
 
 # Disable pi user, with its insecure default password...
 if id -u pi 2> /dev/null; then # DO not show missing user error here...
-	if id -u $ONBOARD_USER > /dev/null; then # ...but so show it here!
-    		echo 'Disabling/locking user "pi" (Raspberry) for security reasons.'
-    		echo "We have the user '$ONBOARD_USER' instead."
-    		passwd -l pi
-	fi
+  echo "Would you like to disable pi user? (y/n)"
+  read answer
+  if [[ $answer == y ]] 2> /dev/null; then
+    if id -u $ONBOARD_USER > /dev/null; then # ...but so show it here!
+      echo 'Disabling/locking user "pi" (Raspberry) for security reasons.'
+      echo "We have the user '$ONBOARD_USER' instead."
+      passwd -l pi
+    fi
+  else
+    echo "Change default password for security reasons"
+    sudo passwd pi
+  fi
+  #clone groups from pi to ONBOARD_USER
```

Comment:
La clonazione dei gruppi da `pi`  a `onboard` secondo me va fatta solo se si elimina `pi` quindi sposterei questa parte di codice sotto l'`if`... Non so se sei d'accordo, parliamone domani o quando possiamo.

Reply:
Non saprei. L'idea era di rendere in ogni caso onboard, se non da solo, pari dignitario di pi insieme a questo. Di fatto, cosi come nasce onboard mi pare non appartenga ai sudoers (anche se questa cosa non mi è molto chiara) quindi ho pensato di mettere il codice per lì per chiudere in ogni caso con la clonazione. Contattami pure su telegram